test/test_device.py

```diff
@@ -583,21 +583,23 @@ def test_signtx(self):
     # Make a huge transaction which might cause some problems with different interfaces
     def test_big_tx(self):
         # make a huge transaction
-        keypool_desc = self.do_command(self.dev_args + ["getkeypool", "--account", "10", "--addr-type", "legacy", "0", "100"])
+        addr_type = "legacy" if self.supports_legacy else "sh_wit"
+        keypool_desc = self.do_command(self.dev_args + ["getkeypool", "--account", "10", "--addr-type", addr_type, "0", "100"])
```

Comment:
It should be fine to just change the address type to `sh_wit` always instead of using `legacy` for the other devices.

Reply:
Okay, changed the types to `sh_wit`